app/src/main/java/com/github/se/orator/ui/speaking/Speaking.kt

```diff
@@ -228,29 +132,29 @@ fun SpeakingScreen(navigationActions: NavigationActions, viewModel: SpeakingView
       }
 }
 
-/**
- * A composable that visualizes audio amplitudes as a waveform.
- *
- * @param amplitudes A list of amplitude values to visualize.
- */
-@Composable
-fun AudioVisualizer(amplitudes: List<Float>) {
-  val color = MaterialTheme.colorScheme.onBackground
-  Canvas(
-      modifier =
-          Modifier.fillMaxWidth()
-              .height(AppDimensions.visualizerHeight)
-              .testTag("audio_visualizer")) {
-        val width = size.width
-        val height = size.height
-        val barWidth = width / amplitudes.size
-        amplitudes.forEachIndexed { index, amplitude ->
-          val barHeight = (amplitude / Short.MAX_VALUE) * height
-          drawLine(
-              color = color,
-              start = Offset(x = index * barWidth, y = height / 2 - barHeight / 2),
-              end = Offset(x = index * barWidth, y = height / 2 + barHeight / 2),
-              strokeWidth = barWidth)
-        }
-      }
-}
+/// **
```

Comment:
I think that you could remove these comments if you don't need the code anymore

Reply:
fixed